app/pages/lab/project-details.cjsx

```diff
@@ -155,15 +155,19 @@ module.exports = React.createClass
 
         <div>
           Data export<br />
-          <button type="button" disabled={@state.exportRequested} onClick={@requestDataExport}>Request data export</button>{' '}
+          <button type="button" disabled={@state.exportRequested} onClick={@requestDataExport}>Request new data export</button>{' '}
           <small className="form-help">
-            CSV format available.{' '}
-            <PromiseRenderer promise={@classificationsExportsGet} then={([mostRecent]) =>
+            CSV format.{' '}
+            <PromiseRenderer promise={@classificationsExportGet}>{([mostRecent]) =>
               if mostRecent?
-                <span>Most recent request was made {moment(mostRecent.created_at).fromNow()}.</span>
```

Comment:
Rails reuses the old model, so created_at won't change. You'll want this to be updated_at to show the most time it was most recently exported.